guacamole-common-js/pom.xml

```diff
@@ -37,6 +37,23 @@
         <relativePath>../</relativePath>
     </parent>
 
+    <properties>
+
+        <!--
+        The location where temporary files should be stored for communicating
+        between karma and firefox. The default location, /tmp, does not work
+        if firefox is installed via snap.
+        -->
+        <firefox.temp.dir>${project.basedir}/.tmp</firefox.temp.dir>
```

Comment:
I think this should go beneath `${project.build.directory}` instead of within the source tree.

Reply:
sure